CHANGES.rst

```diff
@@ -1725,6 +1725,9 @@ astropy.io.ascii
 astropy.io.fits
 ^^^^^^^^^^^^^^^
 
+- Fix a bug in ``io.fits`` with writting Fortan-ordered arrays in Python 3
```

Comment:
```suggestion
- Fix a bug in ``io.fits`` with writting Fortran-ordered arrays in Python 3
```

Reply:
Suggestion for the changelog: *Fix writing Fortran-ordered arrays with a file object* (`nditer` is in all supported versions of Numpy now i think, and the this also applies to Python 2 for v2.0.x)

Reply:
@neutrinoceros , I think this is the remaining review comment to be addressed. And yes, @saimn has the final say in the approval process for this PR. Thank you very much!

Reply:
should be good now !

Reply:
Almost good but please mention that this is only for writing to file objects, and I don't think that it is specific to Python 3 (the PR that changed the method with which file object are written was specific to Python 3, but this bug is not).

Reply:
my bad. Done :)